src/constants.ts

```diff
@@ -6,6 +6,7 @@ export const DEFAULT_SCRIPT_FIRST_TYPE = OutputType.String
 export const KIND_OPTIONS = [Kind.HttpGet, Kind.RNG]
```

Comment:
You should add `Kind.HttpPost` to KIND_OPTIONS
```suggestion
export const KIND_OPTIONS = [Kind.HttpGet, Kind.RNG, Kind.HttpPost]
```